packages/appservice-rest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-actions-appservice-rest",
-  "version": "1.0.8",
+  "version": "1.1.9",
```

Comment:
quite a jump in version ! :p

Reply:
Version in current package.json(1.0.8) was not in sync with the published package (https://www.npmjs.com/package/azure-actions-appservice-rest).
Hence I have upgraded minor version from 1.0.9 to 1.1.9.